documentation/src/utils/framework.ts

```diff
@@ -12,6 +12,10 @@ const frameworks = [
 	{
 		id: "astro",
 		title: "Astro"
+	},
+	{
```

Comment:
Should it be "Qwik" or "QwikCity"?

Reply:
Uhm, the correct name should "Qwik City", but its like a convention to say just Qwik, because qwik comes with qwik-city out of the box